lib/animina/accounts/accounts.ex

```diff
@@ -13,5 +13,6 @@ defmodule Animina.Accounts do
     resource Animina.Accounts.Token
     resource Animina.Accounts.Photo
     resource Animina.Accounts.Reaction
+    resource Animina.Accounts.Message
```

Comment:
I don't know if `Accounts` is the right home for this resource. But we can move that later.